samples/sample_delete_blob.py

```diff
@@ -0,0 +1,28 @@
+from azure.storage.blob import BlobServiceClient, ContainerClient
+import os
+
+def batch_delete_blobs_sample(local_path): 
```

Comment:
Can you put this in a BlobBatchigSamples class?

Reply:
and then definition becomes 
def batch_delete_blobs_sample(self, local_path):

Reply:
Would also need to create and instance and all the method on the instance at the bottom. What's the rationale for complicating the example by adding a class?

Reply:
just consistency for what we already have...

Reply:
FWIW I think having a test to exercise the samples is great but I would not add classes that are not otherwise needed just to satisfy the test framework (might make the tests more complicated but that's the trade-off to keep the samples as focused as possible for users)